### DISCUSSION TITLE
Gaia DR3 stars catalogues in Stellarium

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#3007] [Ideas]
In Windows, part of those catalogs (0-3) are stored in Stellarium/stars/default. The bigger ones (4-8) are stored in user/AppData/Roaming/Stellarium/stars/default, being the biggest 545 MB. 

Cartes du Ciel uses Gaia DR3 stored in Ciel/cat/gaia/gaia1, gaia2 and gaia3, each folder containing 24 subfolders named n9999 plus one gaia.hdr file which I guess carries the heading information. Inside each of those n9999 folders there are hundreds of gaia9999.dat files that serve as tiles to cover the whole sky. In total, 9537 dat files that repeat in each gaia1, 2 and 3 folder to contain higher magnitude stars, so you can install only gaia1 for instance (1,3 GB). Using the three levels, the total number of stars covered is 1.811.709.771 with many fields for each. Obviously, the size then is enormous (64 GB), just as files DE440, 441, 430 and 431 currently usable in Stellarium are (5 GB). https://vizier.cds.unistra.fr/viz-bin/VizieR-4

If coding Stellarium to read Gaia catalogue directly is hard, maybe we could do it the other way round by converting gaia rows into the format in use in Stellarium, as to "cheat" it into believing it's part of its regular default catalog. Direct access though would permit future Gaia data releases to be added with no additional work.

**Comment by gzotti** [comment]
Feel invited!

**Comment by Atque** [comment]
Anyone altering the star catalogs ought to fix the corrupted proper motions of some stars. Cartes du Ciel has accurate proper motion data, for a reference.

**Reply by gzotti** [reply]
My words...
